docs/dashboard-config.md

```diff
@@ -30,14 +30,14 @@ The following are a list of features that are supported, along with there defaul
 | disableKServeMetrics         | false   | Disables the ability to see KServe Metrics.                                                          |
 | disableModelMesh             | false   | Disables the ability to select ModelMesh as a Serving Platform.                                      |
 | disableAcceleratorProfiles   | false   | Disables Accelerator profiles from the Admin Panel.                                                  |
-| disableTrustyBiasMetrics           | false   | Disables Model Bias tab from Model Serving metrics.                                                  |
+| disableTrustyBiasMetrics     | false   | Disables Model Bias tab from Model Serving metrics.                                                  |
 | disablePerformanceMetrics    | false   | Disables Endpoint Performance tab from Model Serving metrics.                                        |
 | disableDistributedWorkloads  | false   | Disables Distributed Workload Metrics from the dashboard.                                            |
 | disableModelRegistry         | false   | Disables Model Registry from the dashboard.                                                          |
-| disableServingRuntimeParams  | false    | Disables Serving Runtime params from the dashboard.                                                  |
-| disableConnectionTypes       | true    | Disables creating custom data connection types from the dashboard.                                   |
-| disableStorageClasses        | false    | Disables storage classes settings nav item from the dashboard.                                       |
-| disableNIMModelServing       | true    | Disables components of NIM Model UI from the dashboard.   
+| disableServingRuntimeParams  | false   | Disables Serving Runtime params from the dashboard.                                                  |
+| disableConnectionTypes       | false   | Disables connection types from the dashboard.                                                        |
```

Comment:
Can you clarify this is not a valid property for the CR but is under the hood? I feel it's odd to have it here if it's not clarified more.

Reply:
@andrewballantyne sorry that should have been removed after I changed my mind to completely remove it from the CRD. Let me update